signer/src/stacks/contracts.rs

```diff
@@ -941,6 +970,60 @@ impl WithdrawalErrorMsg {
     }
 }
 
+/// The responses for validation of a reject-withdrawal-request smart
+/// contract call transaction.
+#[derive(Debug, thiserror::Error, PartialEq, Eq)]
+pub enum WithdrawalRejectErrorMsg {
+    /// The bitmap set in the transaction object should match the one in
+    /// our database.
+    #[error("bitmap does not match expected bitmap from")]
+    BitmapMismatch,
+    /// The smart contract deployer is fixed, so this should always match.
+    #[error("the deployer in the transaction does not match the expected deployer")]
+    DeployerMismatch,
+    /// We do not have a record of the withdrawal request in our list of
+    /// pending and accepted withdrawal requests.
+    #[error("no record of withdrawal request in pending and accepted withdrawal requests")]
+    RequestMissing,
+    /// No bitcoin chain tip
+    #[error("No bitcoin chain tip")]
+    NoBitcoinChainTip,
+    /// Missing bitcoin block
+    #[error("Missing bitcoin block")]
+    MissingBitcoinBlock,
+    /// Missing stacks block
+    #[error("Missing stacks block")]
+    MissingStacksBlock,
+    /// No withdrawal signers
+    #[error("No withdrawal signers")]
+    NoWithdrawalSigners,
+    /// No withdrawal report
+    #[error("No withdrawal report")]
+    NoWithdrawalReport,
+    /// Withdrawal request fulfilled
+    #[error("Withdrawal request fulfilled")]
+    WithdrawalRequestFulfilled,
+    /// Withdrawal request unconfirmed
+    #[error("Withdrawal request unconfirmed")]
+    WithdrawalRequestUnconfirmed,
+    /// Withdrawal request not final
+    #[error("Withdrawal request not final")]
+    WithdrawalRequestNotFinal,
+    /// Withdrawal request wasn't rejected by (num_signers - threshold + 1) signers
+    #[error("Withdrawal request not rejected")]
+    WithdrawalRequestNotRejected,
```

Comment:
No need for Withdrawal in the name of the variants, since the enum is named `WithdrawalRejectErrorMsg`.

Reply:
Done